coding-for-reusability/Exercise4/problem/Problem.tsx

```diff
@@ -5,9 +5,10 @@ Constrain the function type to ensure that the length exists
 */
 
 function getLength<Type>(list: Type) {
-  return Array.isArray(list); //should be returning length
+  return list.length;
 }
 
+//@ts-expect-error
```

Comment:
Oh this is interesting. Maybe we could use a linter if we have these comments in the right places.

Does this show up in CodeSandbox as you’d expect?

Reply:
I should have actually removed that.